tailwind.config.js

```diff
@@ -3,15 +3,24 @@ module.exports = {
   content: ['./src/**/*.{js,jsx,ts,tsx}'],
   theme: {
     extend: {
-      colors :{
-        "primary": "var(--primary)",
-        "secondary": "var(--secondary)",
-        "accent": "var(--accent)",
-        "dark-gray": "var(--dark-gray)",
+      colors: {
+        primary: 'var(--primary)',
+        secondary: 'var(--secondary)',
+        accent: 'var(--accent)',
+        'dark-gray': 'var(--dark-gray)',
+      },
+      fontFamily: {
+        inter: ['Inter'],
+      },
+      width: {
+        500: '500px',
+        300: '300px',
+      },
+      screens: {
+        small: { min: '360px', max: '650px' },
+        medium: { min: '651px', max: '1023px' },
+        large: { min: '1024px', max: '1279px' },
```

Comment:
Superb work on adding these screen sizes here!🙌
This is going to help the whole team use these right away!🔥

Reply:
Thank you Allan